pytorch_lightning/__init__.py

```diff
@@ -24,6 +24,9 @@
     # We are not importing the rest of the scikit during the build
     # process, as it may not be compiled yet
 else:
+    import logging
+    _logger = logging.getLogger("lightning")
```

Comment:
```suggestion
    from logging import getLogger
    _logger = getLogger("lightning")
```
we had some problem with this importing logging in past